### PR TITLE
Allow node to start without build-info.properties information

### PR DESCRIPTION
**Pull request by diega**
build -> *fed:fix_build_info*

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/core/RskFactory.java [hunk 1]
```diff
@@ -415,7 +415,13 @@ public TransactionGateway getTransactionGateway(
     public BuildInfo getBuildInfo(ResourceLoader resourceLoader) throws IOException {
         Properties props = new Properties();
         Resource buldInfoFile = resourceLoader.getResource("classpath:build-info.properties");
-        props.load(buldInfoFile.getInputStream());
+        try {
```
Comment (tinchou) [inline rskj-core/src/main/java/co/rsk/core/RskFactory.java]: This is the 🥔, great work with the previous refactoring. Please remove the now unnecessary exception!

File: rskj-core/src/main/java/co/rsk/core/RskFactory.java [hunk 2]
```diff
@@ -415,7 +415,13 @@ public TransactionGateway getTransactionGateway(
...
+            props.load(buldInfoFile.getInputStream());
+        } catch (IOException ioe) {
+            logger.warn("build-info.properties file missing from classpath");
```
Comment (tinchou) [inline rskj-core/src/main/java/co/rsk/core/RskFactory.java]: Nice logging pattern.

File: rskj-core/src/main/java/org/ethereum/util/BuildInfo.java [hunk 3]
```diff
@@ -22,36 +22,21 @@
...
-            buildBranch = props.getProperty("build.branch");
-        } catch (IOException e) {
-            logger.error("Error reading /build-info.properties", e);
-        }
+    public String getBuildHash() {
+        return buildHash;
     }
 
-    public static void printInfo(){
+    public void printInfo(Logger logger){
```
Comment (tinchou) [inline rskj-core/src/main/java/org/ethereum/util/BuildInfo.java]: Could this be a private logger?

Comment (diega) [reply rskj-core/src/main/java/org/ethereum/util/BuildInfo.java]: I found this method so superfluous that I wanted to give it _some_ kind of meaningfulness. This way you can log in the context of the caller and thus avoid handling an additional Logger in your configuration

File: rskj-core/src/main/java/co/rsk/core/RskFactory.java (hunk 2 above)
Comment (diega) [reply rskj-core/src/main/java/co/rsk/core/RskFactory.java]: Honestly I'm not quite sure this is the better way. I think you can attach different pattern layouts through Logback configuration according the [top] level enabled but I didn't investigate further. We have a deep debt on how we are handling logging, we can add this as an improvement path when we face it

**Pull request merged**